wen-gong/week4/sinatra-servers/movie/views/moviesresults.erb

```diff
@@ -0,0 +1,3 @@
+
+<h2>Movies data for "<%= @exact_movie %>"</h2>
+<% @movies_array.each do|"title"| %>
```

Comment:
there is no `@movie_array` variable in server.rb?
Im assuming you haven't finished this? there needs to be a `<% end %>` for every `do`